content/billing/core-billing/buy-core.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Buy Docker Core
+description: Learn how to start your Docker Core subscription.
+keywords: payments, billing, subscription, invoices, docker core, buy docker
+---
+
+Docker Core subscriptions offer features and benefits to support both new and professional developers, as well as plans for individuals, teams, and enterprise businesses. To learn more about what's included with each plan, see [Docker Core subscriptions and features](../../subscription/core-subscription/details.md) and [Docker Pricing](https://www.docker.com/pricing/) to compare plans.
+
+In this section, learn how to get started with a Docker Core subscription for individuals or for organizations. Before you begin, make sure you have a [Docker ID](../../docker-id/_index.md).
+
+## Buy Docker Core for personal account
+
+1. Sign in to your Docker account.
+2. Select your avatar in the top-right corner then select **Billing**.
+3. In **Billing Details**, select your personal account.
+4. Select **Buy now**.
+5. To continue to use [**Docker Core Personal**](/subscription/core-subscription/details/#docker-personal), select **Continue with Personal**. To buy [**Docker Core Pro**](/subscription/core-subscription/details/#docker-pro), select **Buy Now** on the **Pro** card.
+6. Enter your billing information, then select **Submit**.
+7. Follow the on-screen instructions to confirm monthly or annual billing cycle and enter your payment information.
+8. Select **Subscribe**.
+
+This routes you back to the **Billing Details** section. Your subscription entitles you right away to the features available for your plan. You can review your plan here at any time.
+
+## Buy Docker Core for organizations
+
+You can buy a Docker Core subscription for a new or existing organization. Only organization owners can manage billing for the organization.
```

Comment:
**question:** Should we use the `subscribe` verb over `buy`? To me, `buy` sounds a little scary because it requires spending money, while `subscribe` could be less scary because you might subscribe for free, like with the Docker Personal plan. 

```suggestion
You can subscribe a new or existing organization to a Docker Core plan. Only organization owners can manage billing for the organization.
```